### PR TITLE
[WIP] Post positive test

### PR DESCRIPTION
**Pull request by softvision-oana-arbuzov**
- Added positive tests for petPayload data.
- Updated "CommonApiWrapper" with IntelliJIDEA suggestion (same as for negative tests).

### CODE REVIEW

File: src/main/java/api/utils/CategoryUtils.java [hunk 1]
```diff
@@ -16,4 +16,15 @@ static Category generateDefaultCategory() {
 
         return category;
     }
+
+    static Category generateSpecificCategory() {
```
Comment (ghost) [inline src/main/java/api/utils/CategoryUtils.java]: to keep consistency across project, this should be called `generateDefaultCategory()`

File: src/test/java/api/pet/Post_PositiveTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import api.apiwrappers.PetApiWrapper;
+import api.domain.pets.Pet;
+
+import java.util.ArrayList;
+
+public class Post_PositiveTest {
```
Comment (ghost) [inline src/test/java/api/pet/Post_PositiveTest.java]: Class name should be only camelcase, without underscore: `PostPositiveTest`. Also, we should create packages for positive and negative tests: e.g. `/java/api/positive/pet`

File: src/test/java/api/pet/Post_PositiveTest.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        Assert.assertEquals(postResponsePet.getCategory().getId(), getResponsePet.getCategory().getId(), "The pets's category id is wrong");
+        Assert.assertEquals(postResponsePet.getCategory().getName(), getResponsePet.getCategory().getName(), "The pets's category name is wrong");
+        Assert.assertEquals(postResponsePet.getName(), getResponsePet.getName(), "The pet's name is wrong");
+        Assert.assertEquals(postResponsePet.getTags().get(0).getId(), getResponsePet.getTags().get(0).getId(),"The pets's tag id is wrong");
+        Assert.assertEquals(postResponsePet.getTags().get(0).getName(), getResponsePet.getTags().get(0).getName(),"The pets's tag name is wrong");
+        Assert.assertEquals(postResponsePet.getStatus(), getResponsePet.getStatus(),"The pet's status is wrong");
+    }
+
+    @Test
+    private void VerifyThatAPetIsSuccessfullyCreatedWithSpecificData (){
```
Comment (ghost) [inline src/test/java/api/pet/Post_PositiveTest.java]: Is there a difference between random and specific data (besides that specific data is human readable). If I understand correctly, these two tests do the same thing, with different input data.

File: src/test/java/api/pet/Post_PositiveTest.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        Pet postResponsePet = petApiWrapper.postPet(petPayload);
+
+        //The pet was successfully added
+        assert PetsUtils.checkIfPetsAreEqual(petPayload, postResponsePet);
+
+        //Verify the pet via GET call
+        Pet getResponsePet = petApiWrapper.getPetById(postResponsePet);
+        assert PetsUtils.checkIfPetsAreEqual(postResponsePet, getResponsePet);
+
+        Assert.assertEquals(postResponsePet.getId(), getResponsePet.getId(),"The pets's id is wrong");
```
Comment (ghost) [inline src/test/java/api/pet/Post_PositiveTest.java]: It seems that these set of assertions do the same thing as the one at line 68

File: src/main/java/api/utils/CategoryUtils.java (hunk 1 above)
Comment (softvision-oana-arbuzov) [reply src/main/java/api/utils/CategoryUtils.java]: Removed generateSpecificCategory() method.

File: src/test/java/api/pet/Post_PositiveTest.java (hunk 3 above)
Comment (softvision-oana-arbuzov) [reply src/test/java/api/pet/Post_PositiveTest.java]: True it is the same thing. I removed the one with specific data.

File: src/test/java/api/pet/Post_PositiveTest.java (hunk 4 above)
Comment (softvision-oana-arbuzov) [reply src/test/java/api/pet/Post_PositiveTest.java]: Indeed they do. I removed the duplicate assertions.

File: src/test/java/api/pet/Post_PositiveTest.java (hunk 2 above)
Comment (softvision-oana-arbuzov) [reply src/test/java/api/pet/Post_PositiveTest.java]: Name changed to "PostPositiveTest".

**Pull request closed without merging**